### PR TITLE
Set back libxml_use_internal_errors in test

### PR DESCRIPTION
**Pull request by SpacePossum**
- Revert indent changes in Cookbook

### CODE REVIEW

**Comment by SpacePossum** [comment]
ping @ekho cc @keradus

**Comment by keradus** [comment]
:+1:

File: tests/Report/JunitReporterTest.php [hunk 1]
```diff
@@ -174,7 +174,6 @@ public function testGenerateWithTimeAndMemory()
                         ),
                     )
                 )
-                ->setMemory(2.5 * 1024 * 1024)
```
Comment (ekho) [inline tests/Report/JunitReporterTest.php]: I think we should ensure that memory does not exists in xml

**Comment by merryech0** [comment]
squashed into feature-junit-format

**Pull request merged**